analyses/cell-type-wilms-tumor-06/scripts/run-cnv.R

```diff
@@ -0,0 +1,37 @@
+# Define path -------------------------------------------------------------------
+# The base path for the OpenScPCA repository, found by its (hidden) .git directory
+repository_base <- rprojroot::find_root(rprojroot::is_git_root)
+# The path to this module
+module_base <- file.path(repository_base, "analyses", "cell-type-wilms-tumor-06")
+
+# We build the gene position file reference for infercnv ------------------------
+system(command = glue::glue("Rscript ", file.path(module_base, "scripts", "06a_build-geneposition.R")))
```

Comment:
Let's instead call this directly from the `00_run_workflow.R` script instead. This way, there are no "hidden" steps being called within scripts.